tbx/events/models.py

```diff
@@ -21,8 +16,10 @@ class EventIndexPage(ColourThemeMixin, ContactMixin, SocialFields, Page):
     parent_page_types = ["torchbox.HomePage"]
     subpage_types = []
 
+    events = StreamField([("event", EventItemBlock())], blank=True, use_json_field=True)
```

Comment:
**Comment**: if, at the time of merging this to `main`, #187 would have already been merged, then [`use_json_field=True` could be omitted](https://docs.wagtail.org/en/latest/releases/6.0.html#streamfield-no-longer-requires-use-json-field-true).

Reply:
I'll keep an eye out, thanks Victor.

Note to self: Keep thread unresolved until we're ready to merge the PR.

Reply:
I'm going to go ahead and merge this - agreed with Victor that he'll update the wagtail 6 PR once this is merged.